examples/structured_2d_dgsem/elixir_advection_smview.jl

```diff
@@ -0,0 +1,120 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Coupled semidiscretization of two linear advection systems using converter functions
+# and mesh views for the semidiscretizations. First we define a parent mesh
+# for the entire physical domain, then we define the two mesh views on the parent.
+#
+# In this elixir, we have a square domain that is divided into a left and right.
+# On each half of the domain, a completely independent SemidiscretizationHyperbolic
+# is created for the linear advection equations. The two systems are coupled in the
+# x-direction.
+# For a high-level overview, see also the figure below:
+#
+# (-1,  1)                                   ( 1,  1)
+#     ┌────────────────────┬────────────────────┐
+#     │    ↑ periodic ↑    │    ↑ periodic ↑    │
+#     │                    │                    │
+#     │     =========      │     =========      │
+#     │     system #1      │     system #2      │
+#     │     =========      │     =========      │
+#     │                    │                    │
+#     │<-- coupled         │<-- coupled         │
+#     │         coupled -->│         coupled -->│
+#     │                    │                    │
+#     │    ↓ periodic ↓    │    ↓ periodic ↓    │
+#     └────────────────────┴────────────────────┘
+# (-1, -1)                                   ( 1, -1)
+
+advection_velocity = (0.2, -0.7)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+
+# Create DG solver with polynomial degree = 2 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(polydeg = 3)#, surface_flux = flux_lax_friedrichs)
```

Comment:
Polydeg = 3 in implementation but 2 in comment; What about the surface flux comment?

Reply:
Fixed.